### PR TITLE
Fix broken serial comms to Chiron TFT panel

### PR DESCRIPTION
**Pull request by SwiftNick**
The Chiron TFT became unresponsive after the seral comms refactoring.
The panel was getting integer ASCII pairs rather than chars.
I've changed the SendtoTFT helpers to fix this.
Probably should have done it that way in the first place!

### CODE REVIEW

File: Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp [hunk 1]
```diff
@@ -305,7 +305,8 @@ namespace Anycubic {
         SERIAL_EOL();
       #endif
     }
-    TFTSer.println("");
+    TFTSer.write('\r');
+    TFTSer.write('\n');
```
Comment (X-Ryl669) [inline Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp]: Wouldn't `TFTSer.println()` be enough here?

Comment (SwiftNick) [reply Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp]: It used to work ok like that but the recent changes to the core serial comms broke the behaviour and it stopped working.

Comment (X-Ryl669) [reply Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp]: Got it. My question is why you don't use `TFTSer.println();` here as it's doing exactly the same thing as `TFTSer.write(\r); TFTSer.write(\n);` ?

See [here](https://github.com/MarlinFirmware/Marlin/blob/7a027be25594ae79166b2612c17cf0db3aa29ddf/Marlin/src/core/serial_base.h#L123)

Comment (SwiftNick) [reply Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp]: So the empty quotes messed it up?
I'll try it now

Comment (SwiftNick) [reply Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp]: Yes 
`TFTSer.println();`works ok

The original `TFTSer.println("");` doesn't.

Comment (X-Ryl669) [reply Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp]: Strange. I'll double check this.

**Comment by X-Ryl669** [comment]
Looks good to me.

File: Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp (hunk 1 above)
Comment (X-Ryl669) [reply Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp]: Can you send me your firmware.elf with the `println("")` that does not work ? I'd like to check why it did not build as expected.

Comment (SwiftNick) [reply Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp]: Sorry my bad,
I reverted too many changes it was the `write(c)` change in the lines above that caused the issue.
so `println()` or `println("")` are the same.

Comment (X-Ryl669) [reply Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp]: Still it's better to have the version without the string (println()). Thanks for pointing the error.

**Pull request merged**